app/assets/stylesheets/component-mixins/form.scss

```diff
@@ -1,16 +1,65 @@
+$input-padding-x: vr(2);
+$input-padding-y: vr(1);
+
 @mixin form-text-input {
   background-color: rgba($white, 0.25);
   border: solid 0.0625rem $white;
   border-radius: 1.5625rem;
   color: currentColor;
-  font-size: vr(1.1666666);
+  font-size: 0.875rem;
   font-weight: 600;
   line-height: 1.4;
   margin: 0;
-  padding: vr(1) vr(2);
+  padding: $input-padding-y $input-padding-x;
 
   &::-webkit-input-placeholder { color: currentColor; }
   &::-moz-placeholder          { color: currentColor; }
   &:-ms-input-placeholder      { color: currentColor; }
   &:-moz-placeholder           { color: currentColor; }
 }
+
+// Search
+$icon-font-size: 1rem;
+$search-padding-right: calc(#{$icon-font-size} + #{$input-padding-x} + #{vr(1)});
+
+@mixin search {
+  max-width: 400px;
```

Comment:
Pixels ;)

Reply:
Not sure what you're on about - I can't see any pixels... 👀